crates/pop-common/src/manifest.rs

```diff
@@ -97,6 +98,46 @@ pub fn add_crate_to_workspace(workspace_toml: &Path, crate_path: &Path) -> anyho
 	Ok(())
 }
 
+use std::fs;
```

Comment:
This is not used. Run `cargo +nightly clippy` and fix the warnings like this one.